Makefile

```diff
@@ -501,10 +501,8 @@ publish-artifacts: opscenter telekube
 	if [ -z "$(TELE_KEY)" ] || [ -z "$(DISTRIBUTION_OPSCENTER)" ]; then \
 	   echo "TELE_KEY or DISTRIBUTION_OPSCENTER are not set"; exit 1; \
 	fi;
-	$(GRAVITY_BUILDDIR)/tele logout
-	$(GRAVITY_BUILDDIR)/tele login -o $(DISTRIBUTION_OPSCENTER) --token=$(TELE_KEY)
-	$(GRAVITY_BUILDDIR)/tele push $(GRAVITY_BUILDDIR)/telekube.tar
-	$(GRAVITY_BUILDDIR)/tele push $(GRAVITY_BUILDDIR)/opscenter.tar
+	$(GRAVITY_BUILDDIR)/tele push $(GRAVITY_BUILDDIR)/telekube.tar  --hub=$(DISTRIBUTION_OPSCENTER) --token=$(TELE_KEY)
```

Comment:
Have you made sure that publishing still works? It's done by [Gravity-Publish-5.2](https://jenkins.gravitational.io/view/Publishing/job/Gravity-Publish-5.2/) job. Your "Testing Done" section shows an EOF error while pushing, or is that unrelated?

```
Thu Jul 16 22:09:49 UTC Using Hub https://get.gravitational.io:443
Thu Jul 16 22:09:49 UTC Unpacking image into /tmp/tele038517511
Thu Jul 16 22:09:49 UTC Push finished in now
[ERROR]: unexpected EOF
```

Reply:
I believe the EOF happens because I was not uploading an application tarball, I was uploading "hello word!".

I can run `make -C e clean production publish`, but I was worried about cluttering up the official artifact repo with meaningless artifacts.  I don't know if I have the permissions to delete things once they've been uploaded

Reply:
Upload works:
```
/home/walt/git/gravity/e//build/7.1.0-alpha.1.137/tele push /home/walt/git/gravity/e//build/7.1.0-alpha.1.137/telekube.tar  --hub=https://get.gravitational.io --token=<redacted>                                                                                                                                                     
Fri Jul 17 17:49:56 UTC Using Hub https://get.gravitational.io:443                                                                                                                            
Fri Jul 17 17:49:56 UTC Unpacking image into /tmp/tele569059503                                                                                                                               
Fri Jul 17 17:50:03 UTC Uploading telekube:7.1.0-alpha.1.137 to get.gravitational.io 
        Still uploading telekube:7.1.0-alpha.1.137 to get.gravitational.io (10 seconds elapsed)                                                                                               
// snip ...
        Still uploading telekube:7.1.0-alpha.1.137 to get.gravitational.io (19 minutes elapsed)                                                                                               
Fri Jul 17 18:09:25 UTC Image telekube:7.1.0-alpha.1.137 uploaded to get.gravitational.io                                                                                                     
Fri Jul 17 18:09:25 UTC Push finished in 19 minutes                                                                                                                                           
/home/walt/git/gravity/e//build/7.1.0-alpha.1.137/tele push /home/walt/git/gravity/e//build/7.1.0-alpha.1.137/opscenter.tar --hub=https://get.gravitational.io --token=<redacted>                                                                                                                                                  
Fri Jul 17 18:09:25 UTC Using Hub https://get.gravitational.io:443                                                                                                                            
Fri Jul 17 18:09:25 UTC Unpacking image into /tmp/tele359869776                                                                                                                               
Fri Jul 17 18:09:28 UTC Uploading opscenter:7.1.0-alpha.1.137 to get.gravitational.io                                                                                                         
        Still uploading opscenter:7.1.0-alpha.1.137 to get.gravitational.io (10 seconds elapsed)                                                                                              
Fri Jul 17 18:09:40 UTC Image opscenter:7.1.0-alpha.1.137 uploaded to get.gravitational.io                                                                                                    
Fri Jul 17 18:09:41 UTC Push finished in 16 seconds                                                                                                                                           
make[1]: Leaving directory '/home/walt/git/gravity'                                                                                                                                           
make: Leaving directory '/home/walt/git/gravity/e'   
```

The artifact can be seen here:

https://get.gravitational.io/web/installer/new/gravitational.io/telekube/7.1.0-alpha.1.137

I will delete it after this merges.